cli.js

```diff
@@ -206,6 +211,25 @@ if (options.authentication) {
 	options.authentication = {username, password};
 }
 
+if (options.inset) {
+	const values = options.inset.split(',').map(chunk => Number.parseInt(chunk, 10));
+	const containsNaN = values.findIndex(number => Number.isNaN(number)) > -1;
+
+	if (containsNaN || ![1, 4].includes(values.length)) {
+		console.error('The `inset` flag value is invalid, please refer to the flag examples');
+		process.exit(1);
+	}
+
+	if (values.length === 1) {
+		options.inset = values[0];
+	} else {
+		options.inset = ['top', 'right', 'bottom', 'left'].reduce((acc, key, index) => {
```

Comment:
I prefer to not use `reduce`. Code is usually more readable with a `for-of` loop.